### ISSUE TITLE
Label Encoding of features "Cause" and "Proto"

### ISSUE THREAD

**Issue by SteliosZ** [nthapa-ds/CCD-IDSv1#1]
Hello.

I would like to ask you about those two features. 

I noticed that Argus exports those fields as symbolic values, not continuous.
However, you do not document the process of those features being encoded.

Was it just some universal "LabelEncoder().fit_transform()" function, or are they explicitly encoded ?
How can we know about the corresponding label of each integer in these fields ?

Thank you in advance for your time :)
